caravel/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -98,7 +113,13 @@ class QueryTable extends React.Component {
           />
         </div>
       );
-
+      q.querylink = (
+        <div style={{ width: '75px' }}>
```

Comment:
i think we should let the button be full-width/on one line, and make the sql-query container a little less wide.